Ginger/GingerCoreCommon/GeneralLib/General.cs

```diff
@@ -224,5 +224,15 @@ public static void ClearDirectoryContent(string DirPath)
             }
         }
 
+        public static string RemoveInvalidFileNameChars(string Item)
+        {
+            string fileName = Item;
+            foreach (char invalidChar in Path.GetInvalidFileNameChars())
+            {
+                fileName = fileName.Replace(invalidChar.ToString(), "");
+            }
+            fileName = fileName.Replace(@".", "");
```

Comment:
Are we passing the filename with extension e.g. Activity>111.xml to this method or just the name without extension.

Because if passing with extension then removing the dot will cause problem

Reply:
We are passing only filename without extension.